social-links/tailwind.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+	content: [
+		"./pages/**/*.{js,ts,jsx,tsx}",
+		"./components/**/*.{js,ts,jsx,tsx}",
+	],
+	theme: {
+		extend: {
+			transitionProperty: {
+				height: "height",
+			},
+			zIndex: {
+				negetive: "-1",
+				1: "1",
+			},
+			rotate: {
+				20: "20deg",
+				50: "50deg",
+				80: "80deg",
+			},
+			transitionTimingFunction: {
```

Comment:
did you increase the transition time? I told you that the animation should be a bit faster. It should feel snappy.

Reply:
change the code and then let's see.

Reply:
I had kept the transition time same as the original website, updated it now to 1000ms from 3000ms to make it feel snappier.